test/__snapshots__/index.spec.ts.snap

```diff
@@ -106,32 +106,14 @@ exports[`arrayFunc rules > array-func/from-map > fails on an invalid fixture 1`]
 
 exports[`arrayFunc rules > array-func/prefer-array-from > fails on an invalid fixture 1`] = `
 [
-  {
```

Comment:
I reviewed and updated these snapshots one-by-one, but please feel free to review, they should reflect the changes in the configuration correctly